lib/takeaway.rb

```diff
@@ -0,0 +1,13 @@
+class Takeaway
```

Comment:
What does this class represent? Does the name of the class accurately represent that? Can you think of any names that might be more appropriate?